tests/Unit/Evolution/Systems/GrMhd/ValenciaDivClean/Test_ComovingMagneticFieldMagnitude.cpp

```diff
@@ -0,0 +1,20 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/ComovingMagneticFieldMagnitude.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+
+SPECTRE_TEST_CASE("Unit.GrMhd.ValenciaDivClean.ComovingMagneticFieldMagnitude",
```

Comment:
I think you forgot to add the test to the cmake file?

Reply:
Oh no :(